### PR TITLE
SGD8-2568: Add teaser sq double + Fix teaser space/heading + Change grid/img calc

### PR DESCRIPTION
**Pull request by SvenVdv**
## Description
- teaser class spaces fixed and heading changed to h3
- teaser square double added (only first item, which has the .teaser--square__double class)
- dynamic image ratio and grid calculation instead of hard-coded
- fix for dl > dd padding-left

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [x] I have updated the styleguide CHANGELOG accordingly.

### CODE REVIEW

File: components/11-base/grid/_grid.scss [hunk 1]
```diff
@@ -1,36 +1,165 @@
...
-/// @group grid
-/// @access public
-/// @author Gert-Jan Meire
-///
-%container,
-.container {
-  @include clearfix();
-  @include breakpoint($bp-container) {
-    width: $bp-container;
+// Some imports
```
Comment (lennartvava) [inline components/11-base/grid/_grid.scss]: Meer omschrijvende comment plaatsen in volzinnen.

File: components/11-base/grid/_grid.scss [hunk 2]
```diff
@@ -1,36 +1,165 @@
...
+@import "flexbox-grid/variables/grid-variables";
+@import "flexbox-grid/mixins/visibility-mixins";
+@import "flexbox-grid/mixins/grid-mixins";
+
+
+// Grid classes
```
Comment (lennartvava) [inline components/11-base/grid/_grid.scss]:
```suggestion
// Grid classes.
```

File: components/11-base/grid/_grid.scss [hunk 3]
```diff
@@ -1,36 +1,165 @@
...
+
+.align-baseline {
+  align-self: baseline;
+}
+
+.align-stretch {
+  align-self: stretch;
+}
+
+// Mixin to run inside of for loop - creates col/breakpoint classes * Not for developer use *
```
Comment (lennartvava) [inline components/11-base/grid/_grid.scss]:
```suggestion
// Mixin to run inside of for loop - creates col/breakpoint classes * Not for developer use *.
```

File: components/11-base/grid/_grid.scss [hunk 4]
```diff
@@ -1,36 +1,165 @@
...
-    margin: 0 auto;
   }
 
-  padding: 0 1rem;
-  overflow: hidden;
+  @for $i from 1 through $grid-columns {
+    .col-#{$this-prefix}-#{$i} {
+      //flex-basis: calc(100% / $grid-columns * $i);
+      //max-width: calc(100% / $grid-columns * $i);
+
```
Comment (lennartvava) [inline components/11-base/grid/_grid.scss]:
```suggestion
```

File: components/11-base/grid/_grid.scss [hunk 5]
```diff
@@ -1,36 +1,165 @@
...
+  }
+
+  .first-#{$this-prefix} {
+    order: -1;
+  }
+
+  .last-#{$this-prefix} {
+    order: 1;
+  }
+} // Mixin to run inside of for loop - creates col/breakpoint classes
```
Comment (lennartvava) [inline components/11-base/grid/_grid.scss]: Moet deze comment achter de closing bracket staan?

File: components/11-base/grid/flexbox-grid/mixins/_visibility-mixins.scss [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+      }
+
+      @else {
+        display: block;
+      }
+    }
+  }
+
+  @else {
+    @warn "show mixin arg must be one of the existing breakpoints or ranges (#{$breakpoint-all-prefixes})";
```
Comment (lennartvava) [inline components/11-base/grid/flexbox-grid/mixins/_visibility-mixins.scss:109]:
```suggestion
    @warn "Show mixin arg must be one of the existing breakpoints or ranges (#{$breakpoint-all-prefixes})";
```

File: components/11-base/grid/flexbox-grid/mixins/_visibility-mixins.scss [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+  }
+
+  @else if $breakpoint == "xl" {
+    @media #{$breakpoint-xl-up} {
+      display: none;
+    }
+  }
+
+  @else {
+    @warn "hide mixin arg must be one of the existing breakpoints or ranges (#{$breakpoint-all-prefixes})";
```
Comment (lennartvava) [inline components/11-base/grid/flexbox-grid/mixins/_visibility-mixins.scss]:
```suggestion
    @warn "Hide mixin arg must be one of the existing breakpoints or ranges (#{$breakpoint-all-prefixes})";
```

File: components/41-organisms/collection/_collection.scss [hunk 8]
```diff
@@ -1,29 +1,8 @@
-// Teaser gutter differs from gallery gutter, we need to set up decent grid
-// system with different gutters and items spanning multiple columns.
-// @todo Phase out Susy grid and replace variable below.
-$grid-gutter-width: $gutter-width * 1.5;
+// Dynamic system with different gutters and items spanning multiple columns.
+// Calculation at the bottom
```
Comment (lennartvava) [inline components/41-organisms/collection/_collection.scss]:
```suggestion
// Calculation at the bottom.
```

File: components/41-organisms/collection/_collection.scss [hunk 9]
```diff
@@ -32,63 +11,58 @@ ul.grid-5 {
   text-align: center;
 }
 
-ul.grid-5 {
+// Grid-2-1 is an irregular grid with non-equal columns
```
Comment (lennartvava) [inline components/41-organisms/collection/_collection.scss]:
```suggestion
// Grid-2-1 is an irregular grid with non-equal columns.
```

File: components/41-organisms/collection/_collection.scss [hunk 10]
```diff
@@ -32,63 +11,58 @@ ul.grid-5 {
...
+    > li {
+      @include phablet {
+        @include col(6);
+      }
     }
   }
 }
 
-ul.grid-4 {
+// Tablet version has a deviating nr of columns for grid 5 and 4
```
Comment (lennartvava) [inline components/41-organisms/collection/_collection.scss]:
```suggestion
// Tablet version has a deviating nr of columns for grid 5 and 4.
```

File: components/41-organisms/filter/_filter.scss [hunk 11]
```diff
@@ -37,7 +37,7 @@
       }
 
       > * + * {
-        margin-left: gutter() *.5;
+        margin-left: $gutter-width *.5;
```
Comment (lennartvava) [inline components/41-organisms/filter/_filter.scss]:
```suggestion
        margin-left: $gutter-width * .5;
```

**Review by lennartvava — CHANGES_REQUESTED** [review]

File: components/11-base/grid/_grid.scss (hunk 5 above)
Comment (SvenVdv) [reply components/11-base/grid/_grid.scss]: Nein, heb t gefixed :)

File: components/11-base/grid/_grid.scss (hunk 1 above)
Comment (SvenVdv) [reply components/11-base/grid/_grid.scss]: Done!

**Review by lennartvava — APPROVED** [review]

**Pull request merged**